kernel/build/config-amd64

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 6.16.7 Kernel Configuration
+# Linux/x86 6.16.9 Kernel Configuration
```

Comment:
We can jump to [v6.17](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tag/?h=v6.17), or do you want to leave it out of next pre-release?